jmix-ui/ui/src/main/java/io/jmix/ui/UiComponentProperties.java

```diff
@@ -165,7 +172,8 @@ public UiComponentProperties(
             @DefaultValue("2") int filterPropertiesHierarchyDepth,
             @DefaultValue("3") int filterColumnsCount,
             @DefaultValue("false") boolean filterShowConfigurationIdField,
-            @DefaultValue("true") boolean filterConfigurationUniqueNamesEnabled
+            @DefaultValue("true") boolean filterConfigurationUniqueNamesEnabled,
+            @DefaultValue("id,version,createTs,createdBy,createdBy,createdDate,updateTs,updatedBy,lastModifiedBy,lastModifiedDate,deleteTs,deletedBy,deletedBy,deletedDate") String filterIncludedSystemProperties
```

Comment:
I included the "standard" system attributes from both Cuba and JMix, just to cover bases.